simpleocr/files.py

```diff
@@ -10,80 +9,165 @@
 GROUND_EXTENSIONS_DEFAULT = GROUND_EXTENSIONS[0]
 
 
-def open_img(path_or_name):
-    """
-    Fuzzy finds a image file given a absolute or relative path, or a name.
-    The name might have no extension, or be in the DATA_DIRECTORY
-    """
-    try_img_ext = functools.partial(try_extensions, IMAGE_EXTENSIONS)
-    data_dir_path = os.path.join(DATA_DIRECTORY, path_or_name)
-    path = path_or_name
-    if not os.path.exists(path):
-        # proceed even when there's no result. ImageFile decides on the exception to raise
-        path = try_img_ext(path_or_name) or try_img_ext(data_dir_path) or path
-    return ImageFile(path)
-
 def try_extensions(extensions, path):
-    """checks if various extensions of a path exist"""
+    """ Checks for various extensions if a path exist if the extension is appended """
     for ext in [""] + extensions:
         if os.path.exists(path + ext):
             return path + ext
     return None
 
 
-class GroundFile(object):
-    """A file with ground truth data about a image (i.e.: characters and their position)"""
-    def __init__(self, path):
+def open_image(path):
+    return ImageFile(get_file_path(path))
+
+
+def get_file_path(path, ground=False):
+    """
+    Get the absolute path for an image or ground file. Valid inputs are:
+    - Relative path with or without extension
```

Comment:
Aren't those options right under there?

    - Relative path with or without extension
    - File in DATA_DIRECTORY with or without extension
    - Absolute path with or without extension

And just a file name is pretty much just a relative path anyway.